users/views.py

```diff
@@ -180,6 +180,16 @@ def request_otp(request):
 @api_view(["POST"])
 @permission_classes([AllowAny])
 def verify_otp(request):
+    if "mobile" not in request.data:
+        return Response(
+            {"detail": "mobile not defined"},
+            status=status.HTTP_400_BAD_REQUEST,
+        )
+    if "otp" not in request.data:
+        return Response(
+            {"detail": "OTP Not Provided"},
+            status=status.HTTP_400_BAD_REQUEST,
+        )
```

Comment:
you could club this together:
```suggestion
    for key in ['mobile', 'otp']:
        if key not in request.data:
            return Response(
                {"detail": f"{key} not provided"},
                status=status.HTTP_400_BAD_REQUEST,
            )
```